src/main/java/edu/iu/terracotta/model/app/Assessment.java

```diff
@@ -45,11 +45,11 @@ public class Assessment extends BaseEntity {
     private boolean autoSubmit;
 
     // if null then the number of submissions is unlimited
-    @Column(name = "num_of_submissions", nullable = true)
+    @Column
     private Integer numOfSubmissions;
 
     // if null then no minimum time between submissions
-    @Column(name = "hours_between_submissions", nullable = true)
+    @Column
```

Comment:
Did you intend to remove the column names from these annotations?

Reply:
Yes. JPA handles this automatically, as it will convert the underscored name to the camel-cased variable name. I think it's cleaner and more concise.

"hours_between_submissions" -> hoursBetweenSubmissions

And the "nullable" attribute is true by default.

I can add them back in if you'd prefer. :)

Reply:
Got it, no I think that's fine